background-processor/src/lib.rs

```diff
@@ -53,13 +53,14 @@ impl BackgroundProcessor {
 	/// [`thread_handle`]'s `join()` method to be able to tell if and when an error is returned, or
 	/// implement `persist_manager` such that an error is never returned to the `BackgroundProcessor`
 	///
-	/// `persist_manager` is responsible for writing out the `ChannelManager` to disk, and/or uploading
-	/// to one or more backup services. See [`ChannelManager::write`] for writing out a `ChannelManager`.
+	/// `persist_manager` is responsible for writing out the [`ChannelManager`] to disk, and/or uploading
+	/// to one or more backup services. See [`ChannelManager::write`] for writing out a [`ChannelManager`].
```

Comment:
Probably don't need to link `ChannelManager` in three places. The one on `write` should be sufficient given the method signature has a `ChannelManager` to click on.

Reply:
There's no harm in them all being links, no? Why filter it?

Reply:
I was thinking that repetitive linking could be distracting while reading the docs. But from a cursory scan of some common Rust docs, it seems this actually common practice. So I'm fine with leaving it as is.

I was also hoping to not have to nit you on going over 100 chars. ;) But maybe they already were. :P